### PR TITLE
Get usage records new input format start date end date (for 4.5 branch)

### PR DESCRIPTION
**Pull request by svscorp**
Port of an improvement from PR: #185 

@bhaisaab @kishankavala Here it is.

### CODE REVIEW

File: server/src/com/cloud/usage/UsageServiceImpl.java [hunk 1]
```diff
@@ -224,11 +224,11 @@ public boolean generateUsageRecords(GenerateUsageRecordsCmd cmd) {
...
         }
         TimeZone usageTZ = getUsageTimezone();
-        Date adjustedStartDate = computeAdjustedTime(startDate, usageTZ, true);
-        Date adjustedEndDate = computeAdjustedTime(endDate, usageTZ, false);
+        Date adjustedStartDate = computeAdjustedTime(startDate, usageTZ);
+        Date adjustedEndDate = computeAdjustedTime(endDate, usageTZ);
 
         if (s_logger.isDebugEnabled()) {
-            s_logger.debug("getting usage records for account: " + accountId + ", domainId: " + domainId + ", between " + startDate + " and " + endDate +
+            s_logger.debug("getting usage records for account: " + accountId + ", domainId: " + domainId + ", between " + djustedStartDate + " and " + adjustedEndDate +
```
Comment (bhaisaab) [inline server/src/com/cloud/usage/UsageServiceImpl.java]: Can you fix the typo here, s/djustedStartDate/adjustedStartDate/ -- this is failing the build.
You may also squash the commits, and do a git push -f with your branch on your repo.

Comment (svscorp) [inline server/src/com/cloud/usage/UsageServiceImpl.java]: Yes, already done. Bad typo.

**Comment by bhaisaab** [comment]
LGTM thanks @svscorp  merging now.

**Comment by bhaisaab** [comment]
@svscorp somehow asfgit failed to close it, please close the PR.

**Comment by svscorp** [comment]
Thanks

**Pull request closed without merging**